gadget/install/export_test.go

```diff
@@ -69,3 +71,33 @@ func MockEnsureNodesExist(f func(dss []gadget.OnDiskStructure, timeout time.Dura
 		ensureNodesExist = old
 	}
 }
+
+// LaidOutVolumeFromGadget takes a gadget rootdir and lays out the
+// partitions as specified. This function does not handle multiple volumes and
+// is meant for test helpers only. For runtime users, with multiple volumes
+// handled by choosing the ubuntu-* role volume, see LaidOutUbuntuVolumeFromGadget
+func LaidOutVolumeFromGadget(gadgetRoot string, model gadget.Model) (*gadget.LaidOutVolume, error) {
```

Comment:
Why not move that to install/partition_test.go, refactor to `mustLayOutVolumeFromGadget(c *C, gadgetRoot string, model gadget.Model)` and `c.Assert()` relevant bits?

Reply:
because this was easier :-) but sure I can do that as well